modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'pdf_generator_service/pdf_client'
+require 'claims_api/v2/disability_compensation_evss_mapper'
+require 'evss_service/base'
+
+module ClaimsApi
+  module DisabilityCompensation
+    class DockerContainerService < ClaimsApi::Service
+      def upload(claim_id)
+        log_service_progress(claim_id, 'docker_service',
+                             'Docker container service started')
+
+        auto_claim = get_claim(claim_id)
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
```

Comment:
again, if errored in the previous job during the synchronous flow we shouldn't need this because we should already have reported back to the consumer the error, and we would have either exited after setting an errored status or continued the flow, and it could only be in PENDING.

https://github.com/department-of-veterans-affairs/vets-api/pull/16778/files#r1605542773